src/components/result-ui/summary.js

```diff
@@ -11,11 +13,25 @@ const FieldValue = ({value}) => {
   );
 };
 
-const FieldRow = ({ fieldPath, fieldValue, crumbPath = [] }) => {
+const FieldRow = ({ fieldPath, fieldValue, crumbPath = [], isRelationship, resourceObject }) => {
   const path = (crumbPath||[]).concat([fieldPath]);
+  const { changeFocus } = useContext(FieldFocusContext);
+  const { include } = useContext(LocationContext);
+  const hasRelated = isRelationship && !isEmpty(resourceObject.getRelated(fieldPath));
+  const showDownLink = isRelationship && hasRelated && checkIncludesPath(include, path);
+
+  const handleClick = () => {
+    changeFocus('focusDown', {field: fieldPath, of: resourceObject});
+  };
+
   return (
     <div>
-      <div className="results__field__path">{path.join(' > ')}</div>
+      <div className="results__field__path_container">
+        <span className="results__field__path">{path.join(' > ')}</span>
+        {showDownLink && <span
+          className="results__field__focus--down" onClick={handleClick}
+        >&isin;</span>}
```

Comment:
extremely subtle button !!

Reply:
:relaxed: I figured you'd be able to make something much prettier than me so I went with something really simple and easy to change.